src/main/java/Tasks/Task.java

```diff
@@ -0,0 +1,36 @@
+package Tasks;
+
+import java.util.ArrayList;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "\u2713" : "\u2718"); //return tick or X symbols
```

Comment:
Maybe consider placing the comments on the line above or on top of the method ?
    public String getStatusIcon() {
        //return tick or X symbols
        return (isDone ? "\u2713" : "\u2718");